tests/sles4sap/sapconf.pm

```diff
@@ -35,6 +35,52 @@ sub check_profile {
       unless ($output =~ /Current active profile: $profile/);
 }
 
+sub run_developers_tests {
+    my $devel_repo = 'https://gitlab.suse.de/AngelaBriel/sapconf-test/repository/master/archive.tar.gz';
```

Comment:
maybe create a test variable and get it from variable , so in case any changes ( e.g. need to run different branch because master is broken )  it will be easier to do so

Reply:
well, in practise we can handle a change in the test code quicker and with less pain than changing variables in buillds